db/migrate/20240801170318_add_funding_type_to_courses.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddFundingTypeToCourses < ActiveRecord::Migration[7.1]
+  def change
+    add_column :course, :funding, :string, default: 'fee', null: false
```

Comment:
The `default: 'fee'` is potentially dangerous. 
@elceebee had a great idea to add a "dummy" default value to make is obvious which records need updating. We could use something like "unknown"

Reply:
How do you feel about adding an index on this column as we'll regularly search based on this value 👌

Reply:
I like @elceebee's idea to use a dummy value here. That will make it clear if the correct value hasn't been used yet. I've gone with `not_set` as the default.

An index also makes sense here 👍🏼 

https://github.com/DFE-Digital/publish-teacher-training/pull/4421/commits/380ced1a7ce8fa9e7011a1f4e33f6d1a817aaecf